packages/@react-spectrum/list/docs/ListView.mdx

```diff
@@ -919,7 +976,152 @@ function DragBetweenListsExample() {
+
+<details>
+  <summary style={{fontWeight: 'bold'}}><ChevronRight size="S" /> Show code</summary>
+
+```tsx example export=true render=false
+function DragIntoListDefaultCopy() {
+  let sourceList = useListData({
+    // Similar configuration as last example...
+    ///- begin collapse -///
+    initialItems: [
+      {id: '1', type: 'file', name: 'Adobe Photoshop'},
+      {id: '2', type: 'file', name: 'Adobe XD'},
+      {id: '3', type: 'file', name: 'Adobe InDesign'},
+      {id: '4', type: 'file', name: 'Adobe AfterEffects'}
+    ]
+    ///- end collapse -///
+  });
+
+  let targetList = useListData({
+    // Similar configuration as last example...
+    ///- begin collapse -///
+    initialItems: [
+      {id: '5', type: 'file', name: 'Adobe Dreamweaver'},
+      {id: '6', type: 'file', name: 'Adobe Fresco'},
+      {id: '7', type: 'file', name: 'Adobe Connect'},
+      {id: '8', type: 'file', name: 'Adobe Lightroom'}
+    ]
+    ///- end collapse -///
+  });
+
+  // Same acceptedDragTypes as DragIntoListExample...
+  ///- begin collapse -///
+  // Append a generated key to the item type so they can only be dragged between these two lists
+  let dragType = React.useMemo(() => `keys-${Math.random().toString(36).slice(2)}`, []);
+  let acceptedDragTypes = [`${dragType}-file`];
+  ///- end collapse -///
+  let {dragAndDropHooks: dragListHooks} = useDragAndDrop({
+    // Same getItems and onDragEnd as DragIntoListExample...
+    ///- begin collapse -///
+    getItems: (keys) => [...keys].map(key => {
+      let item = sourceList.getItem(key);
+      // Setup the info for each item that will be propagated upon drag
+      return {
+        [`${dragType}-${item.type}`]: JSON.stringify(item),
+        'text/plain': item.name
+      };
+    }),
+    onDragEnd: (e) => {
+      let {
+        dropOperation,
+        keys
+      } = e;
+      if (dropOperation === 'move') {
+        sourceList.remove(...keys);
+      }
+    }
+    ///- end collapse -///
+  });
+
+  let {dragAndDropHooks: dropListHooks} = useDragAndDrop({
+    onInsert: async (e) => {
+      // Similar onInsert as DragIntoListExample...
+      ///- begin collapse -///
+      let {
+        items,
+        target
+      } = e;
+      ///- end collapse -///
+      /*- begin highlight -*/
+      let processedItems = await Promise.all(
+        items.map(async item => (
+          {...JSON.parse(await item.getText(acceptedDragTypes[0])), id: Math.random().toString(36).slice(2)}
+        ))
+      );
+      /*- end highlight -*/
+      // ...
+      ///- begin collapse -///
+      if (target.dropPosition === 'before') {
+        targetList.insertBefore(target.key, ...processedItems);
+      } else if (target.dropPosition === 'after') {
+        targetList.insertAfter(target.key, ...processedItems);
+      }
+      ///- end collapse -///
+    },
+    onRootDrop: async (e) => {
+      // Similar onRootDrop as DragIntoListExample...
```

Comment:
Since I've kept the examples code hidden in the details, what do you think about just removing the collapse portions within the code? Maybe the highlights are good enough and this way people can just copy paste the example code block without having to fill in the gaps

Reply:
Yeah let's try that.